Listing Plasma Repos as suggested in call #3

tpmccallum opened the pull request:
Listing the different second layer side chain projects which are undergoing design and implementation.
I am just listing the GitHub repos for now, perhaps we could also list papers and documentation if you think that is a good idea.
I am happy to source the content, as required.
This is just a start, I will keep a lookout for others which are under development.

tpmccallum committed:
Listing Plasma Repos as suggested in call 3

Listing the different second layer side chain projects which are undergoing design and implementation.
I am just listing the GitHub repos for now, perhaps we could also list papers and documentation if you think that is a good idea.
I am happy to source the content, as required.
This is just a start, I will keep a lookout for others which are under development.

tpmccallum committed:
Footnote about alphabetical order

The list is in alphabetical order (based on the GitHub account which houses the code).

tpmccallum commented:
@DavidKnott 
Hi David,
Just thought that I would bring this list to your attention.
Really enjoying the implementer calls; thought I would action a good idea from call number 3

luca590 reviewed (approved):
Could we merge this? I think this list is actually quite helpful to have in a repo

tpmccallum committed:
Adding FastX Protocol

One more implementation to add

tpmccallum committed:
Updating the file to link to learnplasma.org

As Learn Plasma mentions, there is a lot of disparate Plasma research, development and testing taking place. It would be a really great service to link this Ethereum Plasma page to the Learn Plasma site.

tpmccallum committed:
Formatting for clarity

colin-axner commented:
Any reasons why this hasn't been merged? This could a valuable repo for developers not sure of the different plasma implementations.

tpmccallum committed:
Adding Kelvin Fichter's MMP repo and remove an old redundant link

tpmccallum committed:
Update with fresh content

colin-axner commented on README.md:
```diff
@@ -1 +1,46 @@
+## FastX Protocol, forked from Omisego's Plasma MVP
+[MVP](https://github.com/FastXProtocol/plasma-mvp)
+
+## Blockchain at Berkeley
+[Plasma Debit](https://github.com/FourthState)
```
```suggestion
## FourthState Labs
[Plasma MVP](https://github.com/FourthState)
```
we didn't do plasma debit, just plasma mvp

tpmccallum committed:
Update README.md

Hi,
Thanks so much!

Co-authored-by: colin axnér <25233464+colin-axner@users.noreply.github.com>
